dv/uvm/riscv_dv_extension/csr_description.yaml

```diff
@@ -62,26 +62,26 @@
       lsb: 0
 
 # MHARTID
-- csr: mhartid
-  description: >
-    Contains integer ID of hardware thread running code
-  address: 0xF14
-  privilege_mode: M
-  rv32:
-    - field_name: cluster_id
-      description: >
-        ID of the cluster
-      type: R
-      reset_val: 0
-      msb: 10
-      lsb: 5
-    - field_name: core_id
-      description: >
-        ID of the core within cluster
-      type: R
-      reset_val: 0
-      msb: 3
-      lsb: 0
+#- csr: mhartid
```

Comment:
Please add a TODO here. After get clarified with the designer, you can add a comment for it.

Reply:
Done